tf-modules/cumulus/ecs_cluster.tf

```diff
@@ -192,21 +192,15 @@ resource "aws_security_group" "ecs_cluster_instance" {
   tags   = var.tags
 }
 
-resource "aws_security_group_rule" "ecs_cluster_instance_allow_ssh" {
-  type              = "ingress"
-  from_port         = 2
-  to_port           = 2
-  protocol          = "-1"
-  cidr_blocks       = ["0.0.0.0/0"]
+resource "aws_vpc_security_group_ingress_rule" "ecs_cluster_instance_allow_ssh" {
+  ip_protocol       = "-1"
```

Comment:
Setting -1 here will allow all protocols on all ports according to

```
[ip_protocol](https://registry.terraform.io/providers/-/aws/latest/docs/resources/vpc_security_group_ingress_rule#ip_protocol-1) - (Required) The IP protocol name or number. Use -1 to specify all protocols. Note that if ip_protocol is set to -1, it translates to all protocols, all port ranges, and from_port and to_port values should not be defined.
```
https://registry.terraform.io/providers/-/aws/latest/docs/resources/vpc_security_group_ingress_rule